### DISCUSSION TITLE
Useful Ruby modules error

### DISCUSSION THREAD

**Discussion by nisreenalnaji** [bassamanator/Sovol-SV06-firmware#74] [General]
Hey there, I've been trying to configure the "Compute the total area of all selected shapes" macro (can be found in this link https://www.klayout.de/useful_scripts.html#calc_area.lym)  but it is not showing in my tools list, can anyone please advise on the steps of configuring this?

**Comment by bassamanator** [comment]
These are Ruby modules. I don't see how you can directly import them into klipper.

**Reply by nisreenalnaji** [reply]
Can you advise on the proper way to configure these into K-layout?

**Comment by bassamanator** [comment]
What are you trying to accomplish, my friend?
